src/NzbDrone.Core/Profiles/Releases/PreferredWordService.cs

```diff
@@ -76,11 +79,29 @@ public List<string> GetMatchingPreferredWords(Series series, string title)
                         matchingPairs.Add(new KeyValuePair<string, int>(matchingTerm, preferredPair.Value));
                     }
                 }
+
+                if (matchingPairs.Count > 0)
+                {
+                    if (!string.IsNullOrEmpty(releaseProfile.Name))
+                    {
+                        var profileName = releaseProfile.Name.Trim();
+                        if (!profileWords.ContainsKey(profileName))
+                        {
+                            profileWords.Add(profileName, new List<KeyValuePair<string, int>>());
+                        }
+
+                        profileWords[profileName].AddRange(matchingPairs);
+                    }
+
+                    if (!profileWords.ContainsKey(string.Empty))
+                    {
+                        profileWords.Add(string.Empty, new List<KeyValuePair<string, int>>());
+                    }
+                    profileWords[string.Empty].AddRange(matchingPairs); // Add the "everything grouping"
```

Comment:
New line after if block.
Don't use a dictionary for the everything bucket, return a new class that has to properties, one that includes everything and one that includes the grouped values.

Reply:
Replaced the Dictionary result with a new PreferredWordMatchResults object. The name feels just a tiny bit off, so if you'd get a better name, I'm all for it.